Database/DBconnect.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose")
+
+const mySecret = process.env['MongoDBCredentials']
+
+const uri = mySecret;
```

Comment:
`mySecret` could be removed, you can just set mongodb credentials to `uri`

Reply:
Ya wrote an extra code. Thanks.